assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -270,16 +270,24 @@ export default function EffectChooser({
 
   const handleUpDown = useCallback(
     ({ key }) => {
-      if (key === 'ArrowUp' && focusedIndex !== 0) {
+      if ((key === 'ArrowUp' || key === 'ArrowLeft') && focusedIndex !== 0) {
         handleMoveFocus(-1);
-      } else if (key === 'ArrowDown' && focusedIndex < listLength - 1) {
+      } else if (
+        (key === 'ArrowDown' || key === 'ArrowRight') &&
+        focusedIndex < listLength - 1
+      ) {
```

Comment:
Does this account for RTL?

I would have expected checking for `isRTL` is needed, perhaps by using `getArrowDir` util

Reply:
Ahh good call. Nope, completely forgot about this. Will add in consideration for RTL. thanks